test/cpp/fluid/mkldnn/test_mkldnn_cpu_quantize_pass.cc

```diff
@@ -0,0 +1,124 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+#include <gtest/gtest.h>
+#include "paddle/fluid/framework/ir/fuse_pass_base.h"
+#include "paddle/fluid/framework/ir/pass.h"
+#include "paddle/fluid/framework/ir/pass_tester_helper.h"
+#include "paddle/fluid/platform/enforce.h"
+#include "paddle/fluid/framework/ir/graph.h"
+#include "paddle/fluid/framework/op_desc.h"
+#include "paddle/fluid/framework/program_desc.h"
+#include <glog/logging.h>
+
+using namespace std;
+
+DEFINE_bool(enable_mkldnn, true, "Enable MKLDNN");
+
+namespace paddle {
+namespace pass {
+
+using VarQuantScale =
+    std::unordered_map<std::string, std::pair<bool, phi::DenseTensor>>;
+    
+static float const SCALE = 2.f;
+const std::vector<std::string> PreGraphPasses({
+   "simplify_with_basic_ops_pass",
+   "quant_dequant_mkldnn_pass",
+   "conv_bn_fuse_pass",
+   "conv_eltwiseadd_bn_fuse_pass",
+   "conv_affine_channel_mkldnn_fuse_pass",
+   "conv_transpose_bn_fuse_pass",
+   "conv_transpose_eltwiseadd_bn_fuse_pass",
+   "conv_bias_mkldnn_fuse_pass",
+   "conv_transpose_bias_mkldnn_fuse_pass",
+   "conv_elementwise_add_mkldnn_fuse_pass",
+   "conv_activation_mkldnn_fuse_pass",
+   "operator_scale_onednn_fuse_pass",
+   "operator_unsqueeze2_onednn_fuse_pass",
+   "operator_reshape2_onednn_fuse_pass",
+   "cpu_quantize_placement_pass",
+   "cpu_quantize_pass",
+});
+
+TEST(cpuQuantizePass, ConvReLU6) {
+  paddle::framework::ProgramDesc prog;
+  auto* block = prog.MutableBlock(0);
+
+  auto* conv2d_op = block->AppendOp();
+  conv2d_op->SetType("conv2d");
+  conv2d_op->SetInput("Input", {"conv2d-X"});
+  conv2d_op->SetInput("Filter", {"conv2d-Y"});
+  conv2d_op->SetOutput("Output", {"conv2d-Out"});
+  
+  const std::vector<int> strides({1, 1});
+  const std::vector<int> paddings({1, 1});
+  const std::vector<int> dilations({1, 1});
+  const int groups = 1;
+
+  conv2d_op->SetAttr("strides", strides);
+  conv2d_op->SetAttr("paddings", paddings);
+  conv2d_op->SetAttr("dilations", dilations);
+  conv2d_op->SetAttr("groups", groups);
+
+  auto* relu6_op = block->AppendOp();
+  relu6_op->SetType("relu6");
+  relu6_op->SetAttr("threshold", 6.f);
+  relu6_op->SetInput("X", {"conv2d-Out"});
+  relu6_op->SetOutput("Out", {"relu-Out"}); 
+
+  auto place = phi::CPUPlace();
+  std::unique_ptr<VarQuantScale> scales(new VarQuantScale());
+  phi::DenseTensor scale_input_tensor;
+  phi::DenseTensor scale_weight_tensor;
+  scale_input_tensor.Resize({1});
+  scale_weight_tensor.Resize({1});
+  auto* ptr_scale_input = scale_input_tensor.mutable_data<double>(place);
+  auto* ptr_scale_weight = scale_weight_tensor.mutable_data<double>(place);
+  ptr_scale_input[0] = SCALE;
+  ptr_scale_weight[0] = SCALE;
+
+  (*scales)["conv2d-X"] = std::make_pair(false, std::move(scale_input_tensor));
```

Comment:
what is the `false` used for?

Reply:
I refer to this implement, it means there is no need to consider the unsigned type. [https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/inference/api/mkldnn_quantizer.cc#L105](url)